docs/appendix/glossary/model-namespace-triplet.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Model Namespace Triplet
+id: model-namespace-triplet
+full_link:
+short_description: namespace:family:name
+aka:
+---
+
+A model is a specific implementation of a resource that implements (speaks) its API.
+Models allow you to control different versions of resource types with a consistent interface.
+
+Models of a {{< glossary_tooltip term_id="resource" text="resource" >}} are uniquely namespaced as colon-delimited-triplets in the form of `namespace:family:name`.
```

Comment:
We already have a glossary entry for "model" so I think this one can be shortened for triplet specificity.
```suggestion
{{< glossary_tooltip term_id="model" text="Models" >}} are uniquely namespaced as colon-delimited-triplets in the form of `namespace:family:name`.
```

Reply:
Awesome, thanks!